src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -118,6 +118,15 @@ public void Execute(TaskExecutionMode mode)
                         this.Differences.Add(diffEntry);
                     }
                 }
+
+                // HashSet contains the set of errors that are stopping the schema compare
+                // Error Message support single message only, so breaking the functionality with single error each time
+                HashSet<string> errorsSet = ComparisonResult.GetAllErrorsList();
+                foreach (string errMsg in errorsSet)
+                {
+                    ErrorMessage = errMsg;
+                    break;
+                }
             }
```

Comment:
Since GetAllErrorsList() function was added in recent DacFx please do not check in in without DacFx update

Reply:
We can - but it scripts correctly and is allowed with space too.

Reply:
sure, I would definitely wait for DacFx merge first!

Reply:
so GetErrors is already present?

Reply:
Yes, GetErrors() is already present in DacFx!